server/controllers/index.js

```diff
@@ -0,0 +1,39 @@
+const { ReviewData } = require('../db/models/Review');
+
+//asyncHandler
+const ah = fn => (req, res, next) => Promise.resolve(fn(req, res, next)).catch(next);
+
+const checkExists = (item, errMess = 'ID does not exist', status = 404) => {
+  if (!item) {
+    let err = new Error(errMess);
+    err.status = status;
+    throw err;
+  }
+}
+
+//GET @  /reviews-api/info/:workspace-id 
+exports.reviewInfo = ah(async (req, res, next) => {
+  const { workspaceId } = req.params;
+  checkExists(workspaceId, 'No valid ID', 400);
```

Comment:
I'm getting a 'Path Does Not Exists' error message when I leave out ID from `notFound`, so I'm not sure there is ever a US case where server responds with `400`..  I don't think this first `checkExists` is actually ever used currently.  Could be good opportunity to write some tests.

Reply:
You're right, it'll never get a hit. Will remove the first calls